app/src/ui/lib/draggable.tsx

```diff
@@ -84,6 +91,10 @@ export class Draggable extends React.Component<IDraggableProps> {
    * just clicking a draggable element.
    */
   private onMouseMove = (moveEvent: MouseEvent) => {
+    if (this.hasMouseUpOccurred) {
+      this.onDragEnd()
+      return
+    }
```

Comment:
Is this necessary? Right now `document.addEventListener('mousemove', this.onMouseMove)` is invoked _after_ the `await sleep(100)`

Reply:
I didn't think so... but it seems to help. :/

Reply:
It was the first thing I did to basically say if mouse move event is happening after mouse down, then close it out. Then I did the debounce thing and thought I shouldn't need it.. but was still able to reproduce.. so I added it back and couldn't reproduce again.. But.. that said it was hard to reproduce anyways.

Reply:
I think it is because the 100ms is just delaying the bug timing. Thus, the bug could still be there if you hold your mouse right around 100ms before doing another click.